sirepo/pkcli/admin.py

```diff
@@ -64,6 +64,28 @@ def create_examples():
                     if example.models.simulation.name not in names:
                         _create_example(example)
 
+# TODO(e-carlin): more than uid (ex email)
+def delete_user(uid):
+    """Delete a user and all of their data across Sirepo and Jupyter
+
+    This will delete information based on what is configured. So configure
+    all service (jupyterhublogin, email, etc.) that may be relevant. Once
+    this script runs all records are blown away from the db's so if you
+    forget to configure something you will have to delete manually.
+    """
+    import sirepo.server
+    import sirepo.template
+    sirepo.server.init()
+    with auth_db.session_and_lock(), \
+         auth.set_user_outside_of_http_request(uid):
+        if sirepo.template.is_sim_type('jupyterhublogin'):
+            from sirepo.sim_api import jupyterhublogin
+            jupyterhublogin.delete_user_dir(uid)
+        pkio.unchecked_remove(simulation_db.user_path(uid=uid))
```

Comment:
It's really important that uid be not None. Otherwise it will delete the entire database. I think this should be a routine in simulation_db that asserts its argument. Similar to jupyterhublogin.delete_user_dir.

Reply:
9abe57661